lib/session.js

```diff
@@ -147,16 +144,13 @@ module.exports = class Session {
     }
   }
 
-  [sign] () {
-    return cookieSignature.sign(this.sessionId, this[secretKey])
-  }
-
   [hash] () {
     const sess = this
     const str = stringify(sess, function (key, val) {
       // ignore sess.cookie property
       if (this === sess && key === 'cookie') {
-        return
+        // we want `touch` to affect the hash of the session
+        return sess.cookie.expires?.toISOString()
```

Comment:
Otherwise, str might result in `expires: {}`

See:

```
> const { configure: configureStringifier } = require('safe-stable-stringify')
undefined
> const stringify = configureStringifier({ bigint: false })
undefined
> const str = stringify({
...   cookie: {
.....     expires: new Date()
.....   }
... }, function (key, val) {
...   // ignore sess.cookie property
...   if (key === 'cookie') {
.....     // we want `touch` to affect the hash of the session
.....     return val.expires
.....   }
... 
...   return val
... })
undefined
> console.log(str);
{"cookie":{}}
undefined
```

Which breaks some of the `saveUninitialized` stuff

Reply:
Should  we maybe serialize it to number by doing .getTime() instead of toISOString? Well i would prefer that.

Reply:
For the purpose of hashing, that makes sense to me!

Reply:
Good call

Reply:
```suggestion
        // we want `touch` to affect the hash of the session
        return sess.cookie.expires?.getTime?.()
```

Reply:
Why would you do the getTime?.

Reply:
Oops, good point.

When some of the previous tests were passing numbers as `cookie.expires`, I was needing to do that. Now that they're fixed, we can avoid the `?.`